core/net/ipv6/uip-nd6.c

```diff
@@ -518,7 +518,7 @@ na_input(void)
         goto discard;
       }
 
-#if NETSTACK_RADIO != ble_mode
+#if NETSTACK_RADIO != ble_mode_driver
```

Comment:
I fixed the typo in the #if check.

I agree that this #if should not be in the official Contiki repo. Does anyone have any suggestions or recommendations on how to create such a generic API to disable the neighbor table functionality?